api/search-tags.md

```diff
@@ -20,12 +20,16 @@ GET
  * `query`               : See [Tag Queries](/tag-queries.md) for more info on tag queries.
  * `activity_start_secs` : (optional) The start time from which to pull data, represented in seconds since the unix epoch.
  * `activity_end_secs`   : (optional) The end time up to which data is pulled, represented in seconds since the unix epoch.
+ * `latest`              : (optional, default 0) Specify if the latest values for the metric should be returned.  Parameters:
```

Comment:
I find this description confusing. I could read it as "the most recent value won't be included in the result set unless I set this", as opposed to what I know it is, "return the latest value as a separate object".  I don't have a good suggestion here; the terminology is just confusing.

Reply:
Huh?  your initial reading is accurate.  The latest values will not be included in the result set unless you set it.

Reply:
OK, then this is MY confusion. I realize now this section is talking about tag searching, so we're not pulling actual metric values anyway.

One question though, is it "values" (plural) or exactly one "latest value"?

Reply:
Last 2 values.